### PR TITLE
Added tests for pages folder

### LINKED ISSUE #166: (SP: ) Increase coverage for pages folder
Increase for `pages` for both mutations and unit tests

### PR DESCRIPTION
**Pull request by heshvab**
Added test for pages. Some of files has no 100 percent because of unready logic which has no point of testing right now.

image
image

### CODE REVIEW

File: code/src/utils/validate-page/validatePage.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const validatePage = (value: unknown) => {
+  return !isNaN(Number(value)) && Number(value) > 0 ? Number(value) : 1;
+};
+
+export default validatePage;
```
Comment (OliaHrytsak) [inline code/src/utils/validate-page/validatePage.ts:5]: Good job!

**Review by OliaHrytsak — APPROVED** [review]

**Review by mhevyk — APPROVED** [review]

File: code/src/pages/orders/OrdersPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import { render, screen } from "@testing-library/react";
+import OrdersPage from "./OrdersPage";
```
Comment (D1nviz) [inline code/src/pages/orders/OrdersPage.test.tsx]: use alias

File: code/src/pages/products/ProductsPage.test.tsx [hunk 3]
```diff
@@ -20,24 +23,135 @@ const mockProducts = [
...
+  });
+};
+
+const renderAndMock = (
+  {
+    entries = "",
+    mockResponse = {}
+  }: {
+    entries?: string;
+    mockResponse?: Partial<RTKQueryReturnState<Partial<typeof mockData>>>;
```
Comment (dmtrth25) [inline code/src/pages/products/ProductsPage.test.tsx:45]: this type `Partial<RTKQueryReturnState<typeof mockData>>` also used for `BestSellers` tests. Can use it once

File: code/src/pages/orders/OrdersPage.test.tsx (hunk 2 above)
Comment (heorhiiss) [reply code/src/pages/orders/OrdersPage.test.tsx]: Thanks, fixed

File: code/src/pages/products/ProductsPage.test.tsx (hunk 3 above)
Comment (heorhiiss) [reply code/src/pages/products/ProductsPage.test.tsx:45]: Thanks, I created a reusable type for that inside of common types

**Review by dmtrth25 — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

**Pull request merged**